rake/spec/default/60-st2_all-services-ok_spec.rb

```diff
@@ -59,13 +59,17 @@
   end
 end
 
-# Check if component services are running
+# Check if component services are running/enabled
 describe 'st2 services' do
   include_examples 'show service log on failure'
 
   spec[:service_list].each do |name|
     describe service(name), prompt_on_failure: true do
       it { is_expected.to be_running }
+      it { should be_enabled.with_level(2) }
+      it { should be_enabled.with_level(3) }
+      it { should be_enabled.with_level(4) }
+      it { should be_enabled.with_level(5) }
```

Comment:
Also, just asking do we really need explicitly to check `2, 3, 4, 5` runlevels?
What is the default for a single `should be_enabled`?

Why I'm asking, because this looks too long in the CircleCI build logs: 
![](https://i.imgur.com/ARy0hYe.png)